flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -302,6 +303,20 @@ public class RocksDBNativeMetricOptions implements Serializable {
                     .withDescription(
                             "Monitor the duration of writer requiring to wait for compaction or flush to finish in RocksDB.");
 
+    public static final ConfigOption<Boolean> MONITOR_DB_GET =
+            ConfigOptions.key("state.backend.rocksdb.metrics.db_get")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "Monitor the metric that measures the time taken for a RocksDB database to perform a read operation in microseconds");
+
+    public static final ConfigOption<Boolean> MONITOR_DB_WRITE =
+            ConfigOptions.key("state.backend.rocksdb.metrics.db_write")
```

Comment:
Same here, I think `state.backend.rocksdb.metrics.db-write` looks better.

Reply:
Why we only chose `db-get` and `db-write` here? MapState would also introduce `db-seek` operations.